tests/e2e/config.go

```diff
@@ -141,3 +148,33 @@ func readConfig(config io.Reader) (e2eTestConfig, error) {
 	err := gcfg.ReadInto(&cfg, config)
 	return cfg, err
 }
+
+// getTestbedConfig returns master node name and IP from K8S testbed
```

Comment:
Can we have this in a seperate file which is exclusively for testbed related utils?